creator-node/src/components/healthCheck/healthCheckComponentService.js

```diff
@@ -50,14 +50,16 @@ const healthCheck = async ({ libs } = {}, logger, sequelize, randomBytesToSign =
  * @param {*} ServiceRegistry
  * @param {*} logger
  */
-const healthCheckVerbose = async ({ libs } = {}, logger, sequelize, getMonitors) => {
+const healthCheckVerbose = async ({ libs } = {}, logger, sequelize, getMonitors, os) => {
   const basicHealthCheck = await healthCheck({ libs }, logger, sequelize)
 
   // Location information
   const country = config.get('serviceCountry')
   const latitude = config.get('serviceLatitude')
   const longitude = config.get('serviceLongitude')
+
   const maxStorageUsedPercent = config.get('maxStorageUsedPercent')
+  const numberOfCPUs = os.cpus().length
```

Comment:
since this doesn't change, can we just cache this upon start up and return that variable? same for the python health check

Reply:
sure, will update

Reply:
Yeah the whole monitors pattern was so that we wouldn't have to do things like this on every request and could pull directly from redis. Slight preference that we use the monitor infra for this too and just set the refresh to daily or something like that. Not super strongly held option though so I'm ok with this as long as we pull out like Dheeraj mentioned